scripts/build_kvm_image.sh

```diff
@@ -38,7 +38,7 @@ prepare_installer_disk()
 {
     fallocate -l 4096M $INSTALLER_DISK
 
-    mkfs.vfat $INSTALLER_DISK
+    mkfs.vfat -n INSTALLER $INSTALLER_DISK
```

Comment:
@liat-grozovik , If it is not added as PR checker, I think it is fine. Mellanox can add their own kvm image.
Can you explain the function of this option and confirm if line:114 works as expected?